app/components/containerConsole/containerConsole.html

```diff
@@ -34,9 +34,16 @@
                       <option value="cmd.exe" ng-if="imageOS == 'windows'">cmd.exe</option>
                     </select>
                   </div>
+                  <input class="form-control" ng-if="state.isCustomCommand" type="text" name="custom-command" ng-model="state.customCommand" placeholder="e.g. ps aux">
```

Comment:
For consistency, could you introduce a new `formValues` object and store `customCommand`, `command` and `user` in this object?